### PR TITLE
Josef's report on PyCon CZ

### PR DESCRIPTION
**Pull request by encukou**
Re-post of Josef Pospíšil's review of PyCon CZ – see [this Twitter conversation](https://twitter.com/damnpepe/status/1005056852575518720).

### CODE REVIEW

File: content/2018-06-11_pyconcz-2018-report.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Title:  PyConCz 2018 — Report
```
Comment (hroncok) [inline content/2018-06-11_pyconcz-2018-report.md:1]: Can we rename it to **PyCon CZ 2018 Report**?

File: content/2018-06-11_pyconcz-2018-report.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Date: 2018-06-11 00:00:00
+Author: Josef Pospíšil
+About: Visit [Josef's blog](https://medium.com/@damnpepe) on Medium.
+
+> This article first appeared on Josef's Medium blog
```
Comment (hroncok) [inline content/2018-06-11_pyconcz-2018-report.md:6]: This note is the thing that get's visible on the fornt page. Can we maybe move it after the first paragraph?

Comment (honzajavorek) [reply content/2018-06-11_pyconcz-2018-report.md:6]: Or maybe at the end?

**Comment by hroncok** [comment]
bump?

**Comment by encukou** [comment]
Maybe just removing the enormous whitespace (`---`) would work?

**Comment by encukou** [comment]
Anyone for a re-review? This repo requires it :)

**Comment by hroncok** [comment]
On it.

**Review by hroncok — APPROVED** [review]
I'd squash.

File: content/2018-06-11_pyconcz-2018-report.md (hunk 1 above)
Comment (hroncok) [reply content/2018-06-11_pyconcz-2018-report.md:1]: @encukou this is till valid BTW but I don't want to block it on it.

**Pull request merged**